pkg/migrations/internal/seq_num.go

```diff
@@ -6,5 +6,5 @@ var (
 	// It is a shared constant between central and the migrator binary.
 	CurrentDBVersionSeqNum = 112
```

Comment:
Is my understanding correct that you don't bump this because you only write postgres migration?